tests/system/providers/amazon/aws/example_ecs.py

```diff
@@ -85,6 +93,7 @@ def get_region():
     asg_name = f"{env_id}-asg"
 
     aws_region = get_region()
+    log_group_name = "/ecs/hello-world"
```

Comment:
I would make this name dynamic (use the env_id as prefix or suffix). Just so we can run twice the same test in parallel

Reply:
oh yeah... I was going to say it's alright because the log streams are different, but since we delete the log group as teardown, it might cause some interferences if there is parallel runs